web/ui/react-app/package.json

```diff
@@ -31,18 +32,20 @@
     "jsdom": "^16.4.0",
     "moment": "^2.24.0",
     "moment-timezone": "^0.5.23",
+    "node-sass": "4.14.1",
     "popper.js": "^1.14.3",
     "react": "^16.7.0",
     "react-copy-to-clipboard": "^5.0.1",
     "react-dom": "^16.7.0",
     "react-resize-detector": "^5.0.7",
     "react-scripts": "3.4.4",
     "react-test-renderer": "^16.9.0",
-    "reactstrap": "^8.0.1",
+    "reactstrap": "^8.9.0",
     "sanitize-html": "^1.20.1",
     "tempusdominus-bootstrap-4": "^5.1.2",
     "tempusdominus-core": "^5.0.3",
-    "typescript": "^3.3.3"
+    "typescript": "^3.3.3",
```

Comment:
it could be nice to move the `typescript` dependency to the `devDependencies` section. And upgrade the typescript to v4 if it's possible. (yeah I'm chasing the dependency that should really be into the dependency section :D)

Reply:
How is that related to the dark theme? I don’t think I follow

Reply:
oh it's not relative, I just saw the diff. So I was more talking about taking the opportunity to do it.

Reply:
I don't think it's a good idea to inflate already large PR with unrelated changes.
I would suggest to open a dedicated PR for it. Especially that it, at least to me, it doesn't seem at all obvious why ( typescript is required to build the app) so a longer explanation there would be useful.

Reply:
Yep let's leave that for a separate PR.